### DISCUSSION TITLE
原本設定的每日排程，在每次更新後都不會被執行 (Trigger issue)

### DISCUSSION THREAD

**Discussion by milestone-git** [AutomaApp/automa#411] [Q&A]
原本設定的每日排程，在每次更新後都不會被執行
trigger by day stop working every time whenever automa has updated

**Comment by Kholid060** [comment]
Are you install Automa from the chrome webstore?

**Reply by milestone-git** [reply]
Yes, I install Automa from the chrome webstore.
